drivers/bus/Kconfig

```diff
@@ -242,4 +242,14 @@ config DA8XX_MSTPRI
 source "drivers/bus/fsl-mc/Kconfig"
 source "drivers/bus/mhi/Kconfig"
 
+config ANCILLARY_BUS
+       tristate "Software based Ancillary Bus"
+       help
+         Provides a software bus for ancillary_devices to be added to it
```

Comment:
example is not accurate.

say:

Provides a bus for matching service purpose. An ancillary device has a unique string based id.
An example is a PCI function which supports multiple class of devices such as RDMA, netdev, vdpa sharing underlying PCI resources among these class drivers.

Reply:
changed